src/core/batch/parquetSchema.ts

```diff
@@ -80,6 +109,39 @@ export const ReactionSchema = {
   signature: { type: "BYTE_ARRAY" },
 };
 
+/**
+ * ReactionBloomFilter: bloom filter options for batching reaction messages
+ */
 export const ReactionBloomFilterOptions = {
   bloomFilters: [{ column: "emoji" }, { column: "fromId" }, { column: "inReplyTo" }],
 };
+
+/**
+ * getSchemaFor() takes DSNPType and returns its corresponding parquet schema
+ *
+ * @param   dsnpType a dsnpType
+ * @returns The corresponding parquet schema
+ */
+export const getSchemaFor = (dsnpType: DSNPType): Schema => {
+  const schemas = [GraphChangeSchema, BroadcastSchema, ReplySchema, ReactionSchema, ProfileSchema];
+
+  return schemas[dsnpType - 1];
+};
+
+/**
+ * getBloomFilterOptionsFor() takes DSNPType and returns its bloom filter options
+ *
+ * @param   dsnpType a dsnpType
+ * @returns The corresponding parquet bloom filter options
+ */
+export const getBloomFilterOptionsFor = (dsnpType: DSNPType): BloomFilterOptions => {
+  const bloomFilterOptions = [
+    GraphChangeBloomFilterOptions,
+    BroadcastBloomFilterOptions,
+    ReplyBloomFilterOptions,
+    ReactionBloomFilterOptions,
+    ProfileBloomFilterOptions,
+  ];
+
+  return bloomFilterOptions[dsnpType - 1];
```

Comment:
Same here. Using a switch case will give type safety to making sure that every DSNPType has a return value